schemas/commands/appeal-final-comment.schema.json

```diff
@@ -0,0 +1,118 @@
+                    "type": ["string", "null"],
+                    "examples": ["I have concerns"]
+                },
+                "submittedDate": {
+                    "type": ["string"],
+                    "format": "date-time",
+                    "examples": ["2023-07-27T20:30:00.000Z"],
+                    "description": "The date the response was submitted by the user"
+                }
+            }
+        },
+        "documents": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "additionalProperties": true,
+                "required": [
+                    "documentId",
+                    "filename",
+                    "originalFilename",
+                    "size",
+                    "mime",
+                    "documentURI",
+                    "dateCreated",
+                    "documentType"
+                ],
+                "properties": {
+                "documentId": {
+                    "type": "string",
+                    "examples": ["4547fec8-16b2-47bb-836d-4d0baac04079"],
+                    "description": "The unique identifier for the document"
+                },
+                "filename": {
+                    "type": "string",
+                    "examples": ["filename.pdf"],
+                    "description": "Current stored name of the document"
+                },
+                "originalFilename": {
+                    "type": "string",
+                    "examples": ["original_filename.pdf"],
+                    "description": "Original name of document"
+                },
+                "size": {
+                    "type": "integer",
+                    "examples": [23452],
+                    "description": "The file size, in bytes"
+                },
+                "mime": {
+                    "type": ["string"],
+                    "examples": ["application/pdf"],
+                    "description": "The mime type for the current version of the file"
+                },
+                "documentURI": {
+                    "type": "string",
+                    "examples": [
+                    "https://custom-domain.gov.uk/container-name/appeal/7002471/4547fec8-16b2-47bb-836d-4d0baac04079/v1/filename.pdf"
+                    ],
+                    "description": "The internal location of the document",
+                    "$comment": "This property contains blob storage info (domain, container), caseReference, version and filename"
+                },
+                "dateCreated": {
+                    "type": "string",
+                    "format": "date-time",
+                    "examples": ["2023-07-27T20:30:00.000Z"],
+                    "description": "The creation date for the document"
+                },
+                "documentType": {
+                    "type": ["string", "null"],
+                    "enum": [
+                        "appellantFinalComments",
```

Comment:
we could do this, or just final comments and attach differently, won't be enough to use rule6FinalComments, as they need to be tied to different users

Reply:
if not clear, this is in regards to using a doc type for all these commands
could either have a doc type per user group: e.g. appeallantFinalComments, lpaFinalComments
or a single doctype: finalComments